src/main/resources/hibernate.cfg.xml

```diff
@@ -6,13 +6,17 @@
 <hibernate-configuration>
     <session-factory>
         <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
-        <property name="connection.url">jdbc:mysql://localhost/cinema?serverTimezone=UTC</property>
+        <property name="connection.url">jdbc:mysql://localhost/cinema?createDatabaseIfNotExist=true&amp;serverTimezone=UTC</property>
         <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
         <property name="connection.username">ma_admin</property>
         <property name="connection.password">1234</property>
         <property name="show_sql">true</property>
         <property name="hbm2ddl.auto">update</property>
```

Comment:
also, will your code work correctly with these properties on your local MySQL server?

Reply:
I put my username and password, added port after DB address, and tested it in the main. The schema was created.